### DISCUSSION TITLE
Progression to Chocofi, Miryoku, Colemak-DH.

### DISCUSSION THREAD

**Discussion by boekhocg** [manna-harbour/miryoku#78] [Show and tell]
![20220301_083510](https://user-images.githubusercontent.com/61322825/156238385-14cf5a9d-f592-4b92-8f7c-4f4f08a15f13.jpg)
![20220301_083911](https://user-images.githubusercontent.com/61322825/156238403-73ed30ec-4df7-43f2-b2f4-9d9e7d85d838.jpg)

Timeline:

October: Was having forearm discomfort. Discovered full split keyboards was a thing and discovered this Sub. Prior to, never used a mechanical keyboard, nor knew full split keyboards existed. Started doing research and falling down the rabbit hole. Decided to go for a 3x6 corne keyboard.

Early November: Learned about Miryoku, home-row mods, and non QWERTY layouts but decided to remain with qwerty and left column shift/ctrl keys to maintain regular keyboard usage experience.

Mid November: Received fully assembled Corne LP. Started practicing and getting used to column stagger. A week later, realized my pinky's do stretch a lot. Decided to switch to homerow-mods using Miryoku keymap. First week was slow typing. By second week, was pretty comfortable.

December: Found a second used Corne LP and purchased it to have a spare and for when/if I go back into an office. Continued practicing typing. Realized I don't need the 6th column at all. Slowly realizing my left hand appears to bet doing a lot more work than the right. Researched alternative layouts further.

January: Switched cold turkey to Colemak-DH. First week or two were very slow typing.

February: Build my first keyboard (chocofi), then build another one with on/off switch since I had extra pcb's. Continue to practice typing and making progress. Very happy with my decision and switch and comfort it brought me. I only wished I had learned this and switched years ago.

TLDR: Switched to Split, Miryoku, Colemak-DH, learned to solder and build my own keyboard. Very happy with the decision to switch and only wished I had learned about and switched years ago. A big thank you to this Sub, [u/manna_harbour](https://www.reddit.com/u/manna_harbour/), creators of QMK and ZMK, and the open source community that makes all of this possible.

P.S.: The magnetic legs were also inspired by an old post on reddit I ran into where manna_harbour posted a fancier version of magnetic legs manna_harbour had made.

**Comment by manna-harbour** [comment]
Thanks for sharing your journey and your builds!  That was a pretty quick progression, I wonder what'll be next?
